uaclient/actions.py

```diff
@@ -249,9 +249,6 @@ def get_cloud_instance(
     try:
         instance = identity.cloud_instance_factory()
     except exceptions.CloudFactoryError as e:
-        if cfg.is_attached:
-            # We are attached on non-Pro Image, just report already attached
-            raise exceptions.AlreadyAttachedError(cfg)
```

Comment:
Why are we removing this bit here ?

Reply:
This check already happens for cli auto-attach at cli.py:1273 and for the api, this PR adds the check at full_auto_attach/v1.py:89